docs/remove-emoji.md

```diff
@@ -0,0 +1,27 @@
+# Remove Emoji
```

Comment:
I'm thinking we should label this as "Remove Emoji Image Replacement".

Removing emoji isn't really what we're doing here. It's a tricky one really, I'm not 100% sure on the best wording...

Reply:
Sounds good to me. I went back and forth on what to put here and fell back to just how the component/plugin was named, but `Remove Emoji Image Replacement` is a lot more clear.

Reply:
Yeah that's better. I'm still thinking about the overall content, it reads a bit like emoji are being disabled which isn't quite right. Need to focus on the image replacement aspect of it more I think.

Reply:
I guess I misunderstood what the module was doing. It's not obvious in the code or inline documentation what the intent is and what's actually happening.